visual-tests/.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules/
+/test-results/
+/playwright-report/
+/playwright/.cache/
+/tests/screenshots/
+/tests/*/snapshots
+/tests/*/screenshots
```

Comment:
I don't know what the structure of the `tests` directory is gonna be, so just checking - is `*` sufficient in this context? Maybe we should use `**` to make sure all the more nested directories will be ignored as well?

Reply:
Actually it is only because I had few versions of paths on local environment (before each test had own directory with subdirectory for screenshots, but it was unnecessary complicated) and wanted avoid to push unnecessary screenshots to repo. These two lines are not necessary anymore.